src/app.js

```diff
@@ -27,28 +27,79 @@ export default class App extends React.Component {
 
   }
 
+  homepage(event) {
+    this.setState({page: 'users'}) // Trying to link back to page of users
+  }
+
   render() {
     if(this.state.page == 'login') {
       return(
           <div className = 'container'>
-            <h1>Connect As: </h1>
-            <div className="btn-group" role = "group">
-            <button type="button" className = "btn btn-secondary" id = "tbutton" onClick = { this.connect }>Tourist</button>
-            <button type="button" className = "btn btn-secondary" id = "gbutton" onClick = { this.connect }>Guide</button>
+            <div className="navbar navbar-default">
+              <div className="container-fluid">
+                <div className="navbar-header">
+                  <p id="logo">Wander</p>
+                </div>
+              </div>
+            </div>
+            <h2 id="title">Welcome to Wander! </h2>
+            <div className = 'col-sm-12 col-md-6'>
```

Comment:
Are you referring the to the "welcome to Wander!" part, or the logo (Wander text on the nav bar)? I'm a little confused as to what you mean because I included customized the css for the Wander text on the nav bar above to be centered